interface/wx/webview.h

```diff
@@ -550,6 +550,9 @@ class wxWebView : public wxControl
         Returns @true if a context menu will be shown on right click.
 
         @since 2.9.5
+        
+        Setting enable to @false will not only disable the showing of the default context menu but 
```

Comment:
This is confusing, it ought to be in `EnableContextMenu()` documentation and not here.